internal/api/photos.go

```diff
@@ -1,14 +1,29 @@
 package api
 
 import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+
 	"github.com/gin-gonic/gin"
 	"github.com/gin-gonic/gin/binding"
+	"github.com/google/uuid"
+	filetype "gopkg.in/h2non/filetype.v1"
+
 	"github.com/photoprism/photoprism/internal/forms"
 	"github.com/photoprism/photoprism/internal/photoprism"
-	"net/http"
-	"strconv"
 )
 
+type PhotosResponse {
+	message: interface{}
```

Comment:
Why is there a `:` after `message`. is this a typo or am i missing something here?.